src/projection/node/create-projection-node.ts

```diff
@@ -399,9 +399,8 @@ export function createProjectionNode<I>({
                         hasLayoutChanged,
                         layout: newLayout,
                     }: LayoutUpdateData) => {
-                        const dragControls = elementDragControls.get(
-                            visualElement
-                        )
+                        const dragControls =
+                            elementDragControls.get(visualElement)
```

Comment:
@mattgperry hmm now the precommit hook starts to introduce these formatting changes, do you have any idea why?

Reply:
No its weird I wonder if Prettier has had a quiet update